spec/features/merchants/items/index_spec.rb

```diff
@@ -128,8 +128,24 @@
         find "img[src='https://kaaskraam.com/wp-content/uploads/2018/02/Gouda-Belegen.jpg']"
       end
     end
+
+    it 'can delete an item' do
+      visit dashboard_items_path
+
+      within "#item-#{@item_1.id}" do
+        expect(page).to have_content(@item_1.id)
+        expect(page).to have_content(@item_1.name)
+        expect(page).to have_content(@item_1.inventory)
+        expect(page).to have_content(@item_1.price)
+
+        expect(page).to have_link('Delete this item')
+        find "img[src='https://kaaskraam.com/wp-content/uploads/2018/02/Gouda-Belegen.jpg']"
+
+        click_link('Delete this item')
+      end
+
+      expect(page).to have_content("Item #{@item_1.id} is now deleted.")
```

Comment:
I like adding in the item number to the message. at some point this may be best to start to do for other things when we refactor.